schema.json

```diff
@@ -0,0 +1,93 @@
+{
+  "$id": "https://raw.githubusercontent.com/public-transport/transport-apis/v1/schema.json#",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "description": "public transport API description",
+  "type": "object",
+  "required": [
+    "name",
+    "type",
+    "supportedLanguages",
+    "coverage",
+    "options"
+  ],
+  "properties": {
+    "name": {
+      "type": "string"
+    },
+    "type": {
+      "type": "object",
+      "propertyNames": {
+        "pattern": "^(efa|hafasMgate|hafasQuery|navitia|otpGraphQl|otpRest)$"
+      },
+      "additionalProperties": {
+        "type": "boolean"
+      }
+    },
+    "supportedLanguages": {
+      "type": "array",
+      "items": {
+        "type": "string",
+        "pattern": "^[a-z][a-z]$",
+        "description": "ISO-639-1 language code"
+      }
+    },
+    "coverage": {
+      "type": "object",
+      "minProperties": 1,
+      "additionalProperties": false,
+      "properties": {
+        "realtimeCoverage": {
+          "$ref": "#coverage"
+        },
+        "regularCoverage": {
+          "$ref": "#coverage"
+        },
+        "anyCoverage": {
+          "$ref": "#coverage"
```

Comment:
The ["structuring a complex schema" guide](https://json-schema.org/understanding-json-schema/structuring.html) uses `#/definitions/coverage`. Shall we use that?

Reply:
Good point. Yes, let's use that.